RNKeychainManager/RNKeychainManager.m

```diff
@@ -248,23 +256,28 @@ - (void)insertKeychainEntry:(NSDictionary *)attributes
   }
 }
 
-- (OSStatus)deletePasswordsForService:(NSString *)service
+- (OSStatus)deletePasswordsForOptions:(NSDictionary *)options
```

Comment:
Does it really make sense to have this as a part of the delete query?

Reply:
@oblador can we clarify this?

Reply:
I think Joel meant if we need the `synchronized` flag for the reset functions, because it doesn't really change behaviour - does it?

Reply:
From my understanding, with the `sychronized` flag the entry is deleted from the iCloud keychain, otherwise in the device keychain